docs/Contributing/adr/0007-pilot-activity-bounded-context.md

```diff
@@ -0,0 +1,385 @@
+# ADR-0007: Pilot activity bounded context
+
+## Status
+
+Proposed
+
+## Date
+
+2025-11-10
+
+## Context
+
+Fleet's Go codebase has grown to over 680,000 lines of code across over 2,300 Go files (across ee/, orbit/, server/, cmd/, and pkg/ directories). We are continuing the transition to a modular monolith architecture by extracting bounded contexts, taking learnings from [ADR-0001 (Android service layer pilot)](0001-pilot-service-layer-packages.md) and evolving the approach.
+
+### Industry examples of modular monolith architecture
+
+**GitLab** (Ruby on Rails, 2.2M+ LOC): Successfully transitioned from monolith to modular architecture. Their [modular monolith documentation](https://handbook.gitlab.com/handbook/engineering/architecture/design-documents/modular_monolith) outlines key motivations: avoiding microservices complexity, maintaining code approachability, and enabling better feature boundaries through DDD. As [GitLab CEO stated](https://about.gitlab.com/blog/why-were-sticking-with-ruby-on-rails/#monorails): "A well-structured, well-architected, highly modular program that runs as a single process."
```

Comment:
Great structure, and GitLab's example of how to transition successfully and transparently.